Gemfile

```diff
@@ -2,7 +2,7 @@
 # If you modify this file manually all comments and formatting will be lost.
 
 source "https://rubygems.org"
-gem "logstash-core", "2.0.0.dev", :path => "."
+gem "logstash-core", "2.0.0.snapshot1"
```

Comment:
@suyograo using  `"~> 2.0.0.snapshot"`  will get everything from now on, imagine we do an snapshot two, etc ... but not beta or other labels (the check is for alphabetic order I think, but not sure) 

using  `"~> 2.0.0.snapshot1"` will do the same.